### PR TITLE
Solitreo: Version 1.000 added (Manual PR)

### PR DESCRIPTION
**Pull request by eliheuer**
Solitreo Version 1.000 taken from the upstream repo https://github.com/ladinoprojects/solitreo at commit https://github.com/ladinoprojects/solitreo/commit/2c1497ba29362c9f5482a408635756f13a6d8860

### CODE REVIEW

File: ofl/solitreo/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+</p>
+<p>
+Solitreo refers to the Hebrew-based cursive script once used by Sephardim; it is the cursive variety of the Rashi alphabet. Solitreo, or Soletreo, is derived from Galician/Portuguese, meaning ‘to spell.’ For many Sephardim, Solitreo was simply known as ganchos, meaning ‘hooks,’ due to the ligatures that form between letters. This style of writing is distinct from the Ashkenazi-based alphabet used for cursive Hebrew today, making documents in Solitreo undecipherable to the untrained eye. A similar style of writing can also be found in documents written in Judeo-Arabic. Today, Solitreo is a relic of the past, as most writers of the language utilize Roman characters.
+</p>
+<p>
+Most non-Hebrew glyphs in this font were forked from <a href="https://github.com/itfoundry/kalam">Kalam</a>.
+</p>
```
Comment (RosaWagner) [inline ofl/solitreo/DESCRIPTION.en_us.html:15]: Missing last sentence "To contribute, see <git url>"

**Comment by RosaWagner** [comment]
Can you re-do the PR with packager so we have an upstream.yaml in the directory?

**Comment by eliheuer** [comment]
Sure, will do that today. Thanks.

File: ofl/solitreo/DESCRIPTION.en_us.html (hunk 1 above)
Comment (eliheuer) [reply ofl/solitreo/DESCRIPTION.en_us.html:15]: This is fixed in commit [322d0a0](https://github.com/google/fonts/commit/322d0a0ee603516498796dbacffa506e101ee832).

**Comment by eliheuer** [comment]
@RosaWagner The packager is having [issues](https://github.com/googlefonts/googlefonts.github.io/issues/55), I'm working on some PRs now to improve the documentation.

But for now, I just force-pushed an update to this PR containing a handwritten `upstream.yaml` file. 

Please let me know if any updates are needed, hopefully you can merge the manual PR with the `upstream.yaml` included. Thanks!

**Comment by RosaWagner** [comment]
I made the PR with packager in #5293

**Comment by eliheuer** [comment]
@RosaWagner I would prefer if I made the PR so I can get credit for the work. Will try to make a PR with the packager today.

**Pull request closed without merging**